geopandas/geodataframe.py

```diff
@@ -1552,35 +1566,69 @@ def explode(self, column=None, **kwargs):
         0  name1  MULTIPOINT (1.00000 2.00000, 3.00000 4.00000)
         1  name2  MULTIPOINT (2.00000 1.00000, 0.00000 0.00000)
 
-        >>> exploded = gdf.explode()
+        >>> exploded = gdf.explode(add_multiindex=True)
         >>> exploded
               col1                 geometry
         0 0  name1  POINT (1.00000 2.00000)
           1  name1  POINT (3.00000 4.00000)
         1 0  name2  POINT (2.00000 1.00000)
           1  name2  POINT (0.00000 0.00000)
 
+        >>> exploded = gdf.explode(add_multiindex=False)
+        >>> exploded
+            col1                 geometry
+        0  name1  POINT (1.00000 2.00000)
+        0  name1  POINT (3.00000 4.00000)
+        1  name2  POINT (2.00000 1.00000)
+        1  name2  POINT (0.00000 0.00000)
+        >>> exploded = gdf.explode(ignore_index=True)
```

Comment:
```suggestion

        >>> exploded = gdf.explode(ignore_index=True)
```

Keep an empty line in between examples.